deepchem/models/torch_models/tests/test_prot_bert.py

```diff
@@ -167,6 +167,9 @@ def test_protbert_overfit():
                               cls_name="FFN",
                               batch_size=1,
                               learning_rate=1e-5)
+    for param in finetune_model.model.bert.parameters():
+        param.requires_grad = False
+
     classification_metric = dc.metrics.Metric(dc.metrics.accuracy_score)
     finetune_model.fit(dataset, nb_epoch=20)
```

Comment:
if params don't have gradients, the model won't fit at all. maybe use a smaller model config to test model fitting

Reply:
I am not really sure what's causing the OOM error, fine-tuning should ideally only fit 1-layer MLP layer.